src/textmate/test/baselines/basic.tokens

```diff
@@ -0,0 +1,227 @@
+//           ~~                                                                                                         string.quoted.single.bicep, punctuation.definition.template-expression.begin.bicep
+//             ~~~~                                                                                                     string.quoted.single.bicep, variable.other.readwrite.bicep
+//                 ~                                                                                                    string.quoted.single.bicep, punctuation.definition.template-expression.end.bicep
+//                  ~~~~                                                                                                string.quoted.single.bicep
+//                      ~~                                                                                              string.quoted.single.bicep, punctuation.definition.template-expression.begin.bicep
+//                        ~~                                                                                            string.quoted.single.bicep, constant.numeric.bicep
+//                          ~                                                                                           string.quoted.single.bicep, punctuation.definition.template-expression.end.bicep
+//                           ~                                                                                          string.quoted.single.bicep
+//                            ~~~                                                                                       string.quoted.single.bicep, constant.character.escape.bicep
+//                               ~~~                                                                                    string.quoted.single.bicep
+//                                  ~~                                                                                  string.quoted.single.bicep, punctuation.definition.template-expression.begin.bicep
+//                                    ~~~~~~~~~~~~~                                                                     string.quoted.single.bicep, variable.other.readwrite.bicep
+//                                                 ~~~~~~~~~~~~~~                                                       string.quoted.single.bicep
+//                                                               ~                                                      string.quoted.single.bicep, punctuation.definition.template-expression.end.bicep
+//                                                                ~                                                     string.quoted.single.bicep
+    properties: {
+//  ~~~~~~~~~~                                                                                                          variable.other.property.bicep
+    }
+  }]
+  
+  resource secrets4 'Microsoft.KeyVault/vaults/secrets@2018-02-14' = [for secret in secretsObject.secrets: if (true) {
```

Comment:
>secret [](start = 74, length = 6)

I think it'd be worth highlighting limitations of TM in these tests as well. For example it'd be ok if `for` was treated as a keyword even if it was also the name of the loop item variable.